include/rppt_tensor_effects_augmentations.h

```diff
@@ -465,6 +465,27 @@ RppStatus rppt_vignette_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t d
 RppStatus rppt_vignette_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32f *vignetteIntensityTensor, RpptROIPtr roiTensorPtrSrc, RpptRoiType roiType, rppHandle_t rppHandle);
 #endif // GPU_SUPPORT
 
+/******************** jitter ********************/
+
+// Jitter augmentation for a NCHW/NHWC layout tensor
+
+// *param[in] srcPtr source tensor memory
```

Comment:
The description needs to be changed as per new format
Please refer to new functions added in this file like vignette and change
https://github.com/r-abishek/rpp/blob/f9a7063d19c06cc9e273353fd1d2b5719cedcaae/include/rppt_tensor_effects_augmentations.h#L425

Reply:
Done